app/views/admin/works/_combined_audio_derivatives.html.erb

```diff
@@ -38,16 +62,48 @@
             <i class="fa fa fa-thumbs-down pr-1" aria-hidden="true"></i>
             Combined audio derivatives exist for this oral history, but they are out of date.
           </p>
+          <% if model&.oral_history_content&.queued? %>
+            <p class="alert-info p-3">
+            A job to recreate them was added to the queue <%=view.time_since_job_status_change %>.
+            </p>
+          <% end %>
+          <% if model&.oral_history_content&.started? %>
+            <p class="alert-info p-3">
+            A job to recreate them started <%=view.time_since_job_status_change %>.
+            </p>
+          <% end %>
+          <% if model&.oral_history_content&.failed? %>
+            <p class="alert-danger p-3">
+            A job to recreate them failed <%=view.time_since_job_status_change %>.
+            </p>
+          <% end %>
           <p>
             <%= link_to "Regenerate combined audio derivatives", create_combined_audio_derivatives_admin_work_path(model), method: "put", class: "btn btn-primary" %>
           </p>
         <% end %>
       <% else %>
         <p>
-            This oral history has <%= view.work_available_members_count %> published audio segment(s).
+            This oral history has <%= view.work_available_members_count %> published audio segment(s). There are no combined audio derivatives for this oral history yet.
         </p>
         <p>
-          There are no combined audio derivatives for this oral history yet.
+          <% if model&.oral_history_content&.queued? %>
+            <p class="alert-info p-3">
+              The combined audio derivatives will be created soon;
+              the job is in the queue as of <%=view.time_since_job_status_change %>.
+            </p>
+          <% end %>
+          <% if model&.oral_history_content&.started? %>
+            <p class="alert-info p-3">
+              They are currently being created;
+              the job started <%=view.time_since_job_status_change %>.
+            </p>
+          <% end %>
+          <% if model&.oral_history_content&.done? %>
+            <p class="alert-danger p-3">
+              Someone tried to create them,
+              but the job failed <%=view.time_since_job_status_change %>.
+            </p>
+          <% end %>
```

Comment:
Wait a THIRD time repeating the same status info? What am I missing?

Reply:
Gone now!